CHANGELOG.asciidoc

```diff
@@ -172,6 +172,7 @@ https://github.com/elastic/beats/compare/v6.2.3...master[Check the HEAD diff]
 - Add support to disable html escaping in outputs. {pull}7445[7445]
 - Refactor error handing in schema.Apply(). {pull}7335[7335]
 - Add additional types to kubernetes metadata {pull}7457[7457]
+- Release `add_docker_metadata` processor as GA. {pull}7497[7497]
```

Comment:
it seems this was done already in https://github.com/elastic/beats/pull/6105 but I forgot the `beta[]` label :man_facepalming: 

Maybe we should remove this line and just delete the `beta[]` flag

Reply:
Removed